### ISSUE TITLE
Failed To Load Synthesizer and Tacotron2 dependencies with Python3.7 and Tensorflow 1.15

### ISSUE THREAD

**Issue by davysouthernboy** [CorentinJ/Real-Time-Voice-Cloning#451]
I have Python3.7 and Tensorflow 1.15, but I get the following when running python3.7 demo_cli.py:

Traceback (most recent call last):
  File "demo_cli.py", line 4, in <module>
    from synthesizer.inference import Synthesizer
  File "/home/davy/Real-Time-Voice-Cloning/synthesizer/inference.py", line 1, in <module>
    from synthesizer.tacotron2 import Tacotron2
  File "/home/davy/Real-Time-Voice-Cloning/synthesizer/tacotron2.py", line 3, in <module>
    from synthesizer.models import create_model
  File "/home/davy/Real-Time-Voice-Cloning/synthesizer/models/__init__.py", line 1, in <module>
    from .tacotron import Tacotron
  File "/home/davy/Real-Time-Voice-Cloning/synthesizer/models/tacotron.py", line 5, in <module>
    from synthesizer.models.modules import *
  File "/home/davy/Real-Time-Voice-Cloning/synthesizer/models/modules.py", line 2, in <module>
    import torch
  File "/usr/local/lib/python3.7/site-packages/torch/__init__.py", line 135, in <module>
    _load_global_deps()
  File "/usr/local/lib/python3.7/site-packages/torch/__init__.py", line 93, in _load_global_deps
    ctypes.CDLL(lib_path, mode=ctypes.RTLD_GLOBAL)
  File "/usr/local/lib/python3.7/ctypes/__init__.py", line 364, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen: cannot load any more object with static TLS

**Comment by blue-fish** [comment]
What is your operating system? You might need to upgrade. This person reported an issue with Ubuntu 14, resolved by upgrading to Ubuntu 16: https://github.com/tensorforce/tensorforce/issues/303#issuecomment-359344743

**Comment by davysouthernboy** [comment]
Ubuntu 14.04.6 LTS.  I'll look into upgrading.  Thank you for your help.

**Comment by mbdash** [comment]
if this can help:
I have Ubuntu 20.

Here is my conda env file:
[rtvc_py373.zip](https://github.com/CorentinJ/Real-Time-Voice-Cloning/files/4978276/rtvc_py373.zip)

Below are the install instruction I wrote down:

**Setup Conda**

DO NOT INSTALL BEFORE READING BELOW : Install for all users

```
cd ~
wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
chmod +x Miniconda3-latest-Linux-x86_64.sh
sudo ./Miniconda3-latest-Linux-x86_64.sh
```

**Install for all users**
To install for all users, change the install path: >>> /opt/miniconda3
```
# Miniconda3 will now be installed into this location:
# /root/miniconda3
#
#   - Press ENTER to confirm the location
#   - Press CTRL-C to abort the installation
#   - Or specify a different location below
[/root/miniconda3] >>> /opt/miniconda3
```

**Fix access rights**
```
cd /opt/
sudo chmod 770 -R miniconda3/
# if using sharing the pachine and part of a user group:
sudo chown -R :your_user_group miniconda3/

exec $SHELL

conda update conda
conda init bash
exec $SHELL
```

**Setup a Conda Virtual Env.**
```
conda env create -f rtvc_py373.yml

# then before starting the demo:
conda activate rtvc_py373

python demo.cli.py
```

**Comment by blue-fish** [comment]
@davysouthernboy Please open a new issue if you have problems setting up the toolbox on a new operating system.
